src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/ConfirmEmployerViewModelValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using SFA.DAS.ProviderCommitments.Web.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Web.Validators
+{
+    public class ConfirmEmployerViewModelValidator : AbstractValidator<ConfirmEmployerViewModel>
+    {
+        public ConfirmEmployerViewModelValidator()
+        {
+            RuleFor(x => x.ProviderId).GreaterThan(0);
+            RuleFor(x => x.Confirm).NotNull().WithMessage("Please select an option");
+        }
+    }
+}
```

Comment:
If this is in place, then we shouldn't need to validate at the controller level. Unless I'm missing something